cmd/convox/templates/init/rails/docker-compose.yml

```diff
@@ -6,3 +6,5 @@ web:
   ports:
     - 80:4000
     - 443:4001
+  volumes:
```

Comment:
All said right now I'm still in favor of volume mounts and hving Convox init inject files into the project. 

Having important stuff already in /app/ in the base image causes so many side effects. Volume mounts Dont worj, and I don't have a good way to copy or customize the bin/web script or rails config file if it doesn't fit my app.

Convox inits job is to look at a code base, and make some guesses about what additional files it needs to run. In the real world every one of these guesses will need to be tweaked. Hard coding these into base images will make that hard.